.circleci/code-sniff.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
+1 to a composer script for this too.

That's what I did in a personal project: https://github.com/stevector/nerdologues-d8/blob/master/composer.json#L80